### PR TITLE
iptables: update to 1.8.6.

### PR DESCRIPTION
**Pull request by abenson**

### CODE REVIEW

File: srcpkgs/iptables/template [hunk 1]
```diff
@@ -12,14 +12,7 @@ maintainer="Anthony Iliopoulos <ailiop@altatus.com>"
...
 homepage="https://www.netfilter.org/"
 distfiles="https://www.netfilter.org/projects/iptables/files/iptables-${version}.tar.bz2"
-checksum=d457d74512e63aa3f50336e0597d4023c0e3c6845594d38532efb6ebcb294309
-	alternatives="
-		iptables:ip6tables:/usr/bin/xtables-legacy-multi
-		iptables:ip6tables-restore:/usr/bin/xtables-legacy-multi
-		iptables:ip6tables-save:/usr/bin/xtables-legacy-multi
-		iptables:iptables:/usr/bin/xtables-legacy-multi
-		iptables:iptables-restore:/usr/bin/xtables-legacy-multi
-		iptables:iptables-save:/usr/bin/xtables-legacy-multi"
```
Comment (ericonr) [inline srcpkgs/iptables/template:22]: Does it create the symlinks by itself, then?

Comment (abenson) [reply srcpkgs/iptables/template:22]: The hell happened there

**Comment by ericonr** [comment]
I had this patch in my local tree as well, it's needed to fix the `ip6tables-apply` dangling symlink.

```diff
diff --git a/srcpkgs/iptables/template b/srcpkgs/iptables/template
index 1103e1da1d..4ce7b5629c 100644
--- a/srcpkgs/iptables/template
+++ b/srcpkgs/iptables/template
@@ -19,7 +19,8 @@ checksum=d457d74512e63aa3f50336e0597d4023c0e3c6845594d38532efb6ebcb294309
 		iptables:ip6tables-save:/usr/bin/xtables-legacy-multi
 		iptables:iptables:/usr/bin/xtables-legacy-multi
 		iptables:iptables-restore:/usr/bin/xtables-legacy-multi
-		iptables:iptables-save:/usr/bin/xtables-legacy-multi"
+		iptables:iptables-save:/usr/bin/xtables-legacy-multi
+		iptables:iptables-apply:/usr/bin/xtables-legacy-multi"
 
 pre_build() {
 	rm include/linux/types.h
@@ -64,5 +65,6 @@ iptables-nft_package() {
 		iptables:ip6tables-save:/usr/bin/xtables-nft-multi
 		iptables:iptables:/usr/bin/xtables-nft-multi
 		iptables:iptables-restore:/usr/bin/xtables-nft-multi
-		iptables:iptables-save:/usr/bin/xtables-nft-multi"
+		iptables:iptables-save:/usr/bin/xtables-nft-multi
+		iptables:iptables-apply:/usr/bin/xtables-nft-multi"
 }
```

**Comment by abenson** [comment]
ip6tables-apply? I don't understand.

**Pull request merged**